zone/exp.cpp

```diff
@@ -202,7 +202,7 @@ uint32 Client::CalcEXP(uint8 conlevel) {
 	return in_add_exp;
 }
 
-uint32 Client::GetExperienceForKill(Mob *against)
+uint64 Client::GetExperienceForKill(Mob *against)
 {
 #ifdef LUA_EQEMU
 	uint32 lua_ret = 0;
```

Comment:
`lua_ret` will need to be adjusted as well for the associated method

```cpp
uint32 LuaParser::GetExperienceForKill(Client *self, Mob *against, bool &ignoreDefault)
{
	uint32 retval = 0;
	for (auto &mod : mods_) {
		mod.GetExperienceForKill(self, against, retval, ignoreDefault);
	}
	return retval;
}
```

This is for the lua mod system

Reply:
Taken care of.